use_cases/timeseries.rst

```diff
@@ -166,7 +167,7 @@ Now whenever maintenance runs, partitions older than a month are automatically d
 Archiving with Columnar Storage
 -------------------------------
 
-Some applications have data logically divides into a small updatable part and a
+Some applications have data logically divided into a small updatable part and a
```

Comment:
Oh, what I meant was they have "data that logically divides." My typo was dropping "that."

Reply:
Will change that.